src/Components/dailyquote/DailyQuote.tsx

```diff
@@ -3,22 +3,33 @@ import "./DailyQuote.css"
 import { DailyQuoteInterface } from "../../interfaces"
 import { fetchDailyQuote } from "../../apiCalls"
 
-const DailyQuote = () => {
+interface DailyQuoteProps {
+  error?: string;
+  setError: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const DailyQuote = ({error, setError}: DailyQuoteProps) => {
   const emptyQuote = {q: "", a: "", h: ""}
   const [ dailyQuote, setDailyQuote] = useState<DailyQuoteInterface>(emptyQuote)
 
   useEffect(() => {
     fetchDailyQuote()
     .then(data => setDailyQuote(data[0]))
+    .catch(error => setError(error.message))
   }, [])
 
   return (
     <div className="daily-quote-wrapper">
-     <div className="daily-top-container">
-        <h2 className="daily-quote">Daily Quote</h2>
-				<h2 className="quote" >"{dailyQuote.q}"</h2>
-				<p className="author" >-{dailyQuote.a}</p>
-			</div>
+      {error && <p>Uh oh! We've encountered an error! {error}</p>}
+      {dailyQuote.a === "zenquotes.io" ? 
```

Comment:
Good call on checking out the response in Postman so we could set up our error handling appropriately! 🎉